### ISSUE TITLE
Installing pipgrip fails

### ISSUE THREAD

**Issue by cbrnr** [ddelange/pipgrip#63]
#### What you were trying to do (and why)
I tried to `pip install pipgrip` on my Mac.

#### What happened (including command output)

Installation failed because of a permission error:

```
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/usr/local/LICENSE'
Consider using the `--user` option or check the permissions.
```

#### What you expected to happen

This error should not occur, because `LICENSE` should not be copied into `/usr/local`. Instead, it should be available in the packages's `site-packages` directory.  User-installing is just a workaround and doesn't fix the problem.

#### Step-by-step reproduction instructions

1. Install Homebrew
2. `brew install python`
3. `pip install pipgrip`

**Comment by ddelange** [comment]
Hi @cbrnr,

Interestingly, no such problems occur on my side:
![image](https://user-images.githubusercontent.com/14880945/133079989-78457daa-c59b-488a-b170-dcd3f7d22105.png)

I doubt it is a packaging issue, as this repo is mostly in line with [pypa/sampleproject](https://github.com/pypa/sampleproject/blob/90d44abe361688aba5a189e661423863b34f5208/MANIFEST.in#L7):
![image](https://user-images.githubusercontent.com/14880945/133080346-71d71b18-2f53-4206-876e-2a66ca592966.png)

What is the output when you run `which pip`? Is it the path to homebrew-python's `pip`? I remember the ``Consider using the `--user` option or check the permissions.`` error when using Mac's builtin system Python to install/run things. I'm pretty sure that this error won't occur when using the pip that comes packaged with homebrew's python :) Maybe homebrew is not properly initialized in your (bash) profile yet? E.g. for homebrew I have these in my [`.bash_profile`](https://github.com/ddelange/new-mac-setup/blob/cbe646adb2131d59cea4c4f486001c17bb69536d/.bash_profile#L1-L3) to prioritize the stuff that brew exposes over any system versions of it (in the default `PATH`):

```
# homebrew
export PATH="/usr/local/sbin:/usr/local/bin:$PATH"
export PATH="${HOME}/.cargo/bin:$PATH"  # rust binary installation path
```

Potentially related: https://github.com/googlesamples/assistant-sdk-python/issues/236

Let me know if that solved it!

**Comment by cbrnr** [comment]
Thanks for your quick reply @ddelange! Nice prompt BTW 😃!

I agree that the project structure is following the guidelines, so this is not the cause of the problem. `pip` is definitely the correct Homebrew version:

```
$ pip -V
pip 21.2.4 from /usr/local/lib/python3.9/site-packages/pip (python 3.9)
```

I've been using Homebrew on my Mac for years, and I've never had a problem like this, so I'm pretty sure I've configured it correctly (in fact, I am using the default config because I have not tinkered with it).

Strangely enough, the error occurs only if the dependencies are not installed. So on a clean slate, running `pip install pipgrip` fails with the permission error. However, running the same command a second time works! I have traced the problem back to the `anytree` package, so this error is really caused by `pip install anytree` because they include the `LICENSE` as a data file (https://github.com/c0fec0de/anytree/blob/master/setup.py#L52). I will open an issue in their repo – sorry for bothering you with this!
